jdaviz/app.vue

```diff
@@ -9,13 +9,18 @@
         </j-tooltip>
       </v-toolbar-items>
       <v-spacer></v-spacer>
+      <v-toolbar-items>
+        <j-tooltip tipid="app-help">
+          <v-btn icon :href="'https://jdaviz.readthedocs.io/en/latest/'+config+'/index.html'" target="_blank">
```

Comment:
Do we need a `v-if` here to make sure the config isn't Specviz2D, since we don't currently have a docs page for that config but people might use it in MAST and try to click through to the docs?

Reply:
that's a good point... the easy fix to both this and latest/stable would be to link to the top-level page, but I think its nice to link to the config-specific landing page.  How about if I add some logic to only go to the config-specific if its within a preset list of supported configs?

Reply:
I think it's nice to send them to the config-specific landing page too, I think your idea is the way to go.